generate/generate_check_test.go

```diff
@@ -23,7 +23,95 @@ import (
 	"github.com/mineiros-io/terramate/test/sandbox"
 )
 
-func TestCheckReturnsOutdatedStackFilenames(t *testing.T) {
+func TestCheckReturnsOutdatedStackFilenamesForExportedTf(t *testing.T) {
+	s := sandbox.New(t)
+
+	stackEntry := s.CreateStack("stacks/stack")
+	stack := stackEntry.Load()
+
+	assertOutdated := func(want []string) {
```

Comment:
non-blocking: maybe `assertOutdatedStacks` would make clearer what we're asserting?

Reply:
since the function is defined inside the test (scope is quite small) I thought a smaller name was OK, but will think on a better name if it was confusing.